sonar-scanner-engine/src/main/java/org/sonar/scanner/report/ReportPublisher.java

```diff
@@ -177,7 +180,7 @@ String upload(File report) {
       .setParam("projectBranch", moduleHierarchy.root().getBranch())
       .setPart("report", filePart);
 
-    if (analysisMode.isIncremental()) {
+    if (analysisFlags.isIncremental()) {
       post.setParam("characteristic", "incremental=true");
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "characteristic" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1192)